backend/README.md

```diff
@@ -1,38 +1,29 @@
-# SSO Pathfinder Confidential Client Using Node-Express
+# URL-S Backend
 
-## Description
+Instructions for configuring and starting the URL-S backend apps: Node.js Express and MongoDB.
 
-This is an example backend app that authenticates users using OAuth 2.0 standard flow
+## Local Setup
+ **`Step 1`**. Make sure you have Node.js installed in your local machine. Type node -v in Terminal. This repo has been tested with Node version 20.10.0 LTS. If you do not have Node, download from here - https://nodejs.org/en/. npm will be installed alongside node.js.
+ 
+ **`Step 2`**. Make sure you have MongoDB installed and running. Install Homebrew and then `brew install mongodb-community@7.0`. Start the database service with `brew services start mongodb-community`
+ 
+ **`Step 3`**. Make sure you have the MongoDB Compass (GUI) app available. This desktop app is used to manage MongoDB databases. Find it at https://www.mongodb.com/try/download/compass. Check that you can connect to the database at localhost:27017 (mongodb://localhost:27017).
+ 
+ **`Step 4`**. Create an .env file to store the BC Government Single Sign-on SSO secret. Copy .env.template to .env. Download the secret from https://bcgov.github.io/sso-requests; click Login; select the URL Shortener project; click Download in INTEGRATION DETAILS; Open the downloaded .json file in a text editor. Copy and paste the secret value into the new .env file. Note that .gitignore lists .env - it will not be added to the repository. 
```

Comment:
It wasn't really clear where to put the .env file. It should be mentioned that it needs to be put under backend folder.

Reply:
Fixed in commit = Update README.md to clarify env file must be created in the backend fold